test/core/test_environ.out

```diff
@@ -3,5 +3,5 @@ LOGNAME=web_user
 PATH=/
 PWD=/
 HOME=/home/web_user
-LANG=(C|en_US).UTF-8
```

Comment:
The main point of these tests is to verify that setting en_US in outer environment is respected by emscripten.

If the `@with_env_modify({'LANG': 'en_US.UTF-8'})` line in the test code doesn't work on windows we should probably look into why its not works and/or disable the test under windows (with a bug open to investigate why it doesn't work).